MetaMask/metamask-extension
# Updating title case to sentence case and other conventions

## Explanation

Currently, we are inconsistent with our content casing conventions across the app.

This is a problem because although subtle it causes inconsistencies across our app and impacts our brand, accessibility and the message we convey to our users.

In order to solve this problem, this pull request updates all EN content to use sentence case with some exceptions and updates some other conventions.

### Exceptions to sentence case
**Names, companies, abbreviations or nouns**
e.g. “MetaMask”, “OpenSea”, “Google Chrome”, “Ledger Live”, “NFT”, “API”

**Special terms**
e.g. “Secret Recovery Phrase”, "Private Key"
"Secret Recovery Phrase' and "Private Key" an extremely important concepts that are critical to preventing users losing their funds. To convey the importance of these terms, they use title case.

**Page navigation**
When referring to the navigation route in the app we quote the page as its title appears e.g.
- "You can enable Ledger Live support by clicking Settings > Advanced > Use Ledger Live.”
- "Go to Settings > Advanced”
- “Go to Settings > Network and enter the chain ID. You can find the chain IDs of most popular networks on $1”

Reference

<img width="450" alt="Screen Shot 2022-07-19 at 1 46 21 PM" src="https://user-images.githubusercontent.com/8112138/179845293-6c367f8a-2272-4590-a5b8-18feccd5ee9b.png">

**Other content**
Other content which may raise questions as to whether they should be capitalized and that should also follow the sentence case convention include:

- Any special MetaMask feature e.g “smart transactions”, “enhanced gas fee”
- “support center”
- “explorer” (as in blockchain explorer)
- ~“mainnet” reason: [https://github.com/ethereum/ethereum-org-website/issues/1882](https://github.com/ethereum/ethereum-org-website/issues/1882)~ Going to open this can of worms in another PR
- “test network” (as in Rinkeby test network”)

## More Information

* Fixes #15039 
[Slack thread](https://consensys.slack.com/archives/G4V2HTG0Y/p1644264432332229)
[Notion doc](https://www.notion.so/Content-Copy-Style-Guide-9dd1a6f2d42f45598dc1f1454abd8688)

## Manual Testing Steps

This touches a lot of content so I would suggest just looking at the code changes to see if the conventions are correct.
- Only `app/_locales/en/messages.json` in locals needs to be thoroughly scrutinized. All other local files are description updates that were updated automatically using `yarn verify-locales:fix`
- Because this PR touches a lot of text and we use text to find nodes for e2e testing there are a lot of e2e test updates.
- No other content should be changed apart from the casing and conventions mentioned above.

## Pre-Merge Checklist


### + If there are functional changes:
